src/devices/Ht1632/Command.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Ht1632
+{
+    /// <summary>
+    /// Command Summary
+    /// </summary>
+    public enum Command : byte
+    {
+        // References middle 8-bits of command codes like:
+        // 100 0000-0000-X
+        //     ~~~~ ~~~~
+
+        /// <summary>
+        /// Turn off both system oscillator and LED duty cycle generator (default)
+        /// </summary>
+        SYS_DIS = 0b_0000_0000,
```

Comment:
For all the public enum and other members, we're using a naming convention that is no C style but C# style. Meaning that this would be without abbreviation when possible, with a capital letter at the beginning of the word, each word after each others without any separation. That will then give for this one: `DisactivateSystemAndLed`

Reply:
Now the question is: do you need to expose this `Command` public? In case of an internal or private enum, and if those are register like you find in the documentation, it's ok to keep them as they are.

Reply:
they are from datasheet P20, just called "Command"

Reply:
names comes entirely from the datasheet

Reply:
Issue here is that they are public. We're allowing them to be like that for internal, not for public. In this case, it's public, so it must be adjusted to more developer friendly naming. If command does not need to be public, make it internal for example and it will be good like this